x-pack/test/functional/apps/aiops/explain_log_rate_spikes.ts

```diff
@@ -156,22 +158,25 @@ export default function ({ getPageObject, getService }: FtrProviderContext) {
         },
       });
 
-      for (const i of [...Array(100)]) {
-        await es.index({
-          index: 'ft_farequote',
-          body: {
-            '@timestamp': '2016-02-09T16:19:59.000Z',
-            '@version': i,
-            airline: 'UAL',
-            custom_field: 'deviation',
-            responsetime: 10,
-            type: 'farequote',
-          },
-        });
-      }
+      await es.bulk({
+        refresh: 'wait_for',
+        body: [...Array(100)].flatMap((i) => {
+          return [
+            { index: { _index: ES_INDEX } },
+            {
+              '@timestamp': '2016-02-09T16:19:59.000Z',
+              '@version': i,
+              airline: 'UAL',
+              custom_field: 'deviation',
+              responsetime: 10,
+              type: 'farequote',
+            },
+          ];
+        }),
+      });
 
       await es.index({
```

Comment:
With the bulk, do we still need this separate index request?

Reply:
Good catch, removed it in [f865360](https://github.com/elastic/kibana/pull/145699/commits/f865360e303dff276b8b06b121eaf07b72177e8a).